github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml

```diff
@@ -467,16 +467,9 @@ postsubmits:
               - |
                 cat "$QUAY_PASSWORD" | podman login --username $(cat "$QUAY_USER") --password-stdin=true quay.io
                 cd images
-                ./publish_image.sh pr-creator quay.io kubevirtci
+                ./publish_multiarch_image.sh pr-creator quay.io kubevirtci
             securityContext:
               privileged: true
-            resources:
-              requests:
-                memory: "16Gi"
```

Comment:
Can you please just re-add the memory requests here ? This just ensures that is gets enough RAM on the node.

Reply:
As this is now a multi-arch cross build on x86 similar to [publish-bootstrap-image](https://github.com/kubevirt/project-infra/blob/5ec81c03232510824d4c163e47038304a3163178/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml#L229-L233), shall I add same resources as defined there ? ie.,

```
             resources:
              requests:
                memory: "29Gi"
              limits:
                memory: "29Gi"
```

Reply:
Yes that should be good